docs-site/content/ko/worktree/guide.md

```
@@ -53,182 +53,156 @@ MoAI-ADK는 이 기능 위에 SPEC 단위의 격리 환경을 얹습니다. SPEC
 ```
 
 ---
 
-### moai worktree go
+### moai worktree sync
 
-Worktree 경로를 출력합니다. 경로 문자열만 표준 출력으로 내보낼 뿐 셸 세션을
-새로 띄우지는 않으므로, `cd`와 묶어서 씁니다.
+Worktree를 base 브랜치의 변경 사항과 동기화합니다.
 
 #### 문법
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)